src/components/structures/MatrixChat.tsx

```diff
@@ -394,7 +395,18 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         }
 
         const crossSigningIsSetUp = cli.getStoredCrossSigningForUser(cli.getUserId());
-        if (crossSigningIsSetUp) {
+
+        // Pulls "default" value for e2e from well-known config.
+        // This can be used to check if encryption is enabled/disabled in home-server.
+        const encryptionIsEnabled = () => {
+            if (!SettingsStore.getValue(UIFeature.LookForWellKnownConfigFromHomeServer)) {
+                return null;
```

Comment:
I don't understand return null ... perhaps u meant return true?
As discussed in the other PR for the matrix-js-sdk, if the UIFeature.LookForWellKnownConfigFromHomeServer is moved into that code, then the check can be done there.

Reply:
This if condition is redundant when check is done in autodiscovery.js so i have removed it now after implementing your suggestion. I think I was trying to tell code to do nothing if config is not found. I am not sure if i was right with that null there though.